pkg/autodiscovery/configresolver/configresolver.go

```diff
@@ -448,11 +448,11 @@ func tagsAdder(tags []string) func(interface{}) error {
 		}
 
 		if typedTree, ok := tree.(map[interface{}]interface{}); ok {
-			tagList, _ := typedTree["tags"].([]string)
+			tagList, _ := typedTree["tags"].([]interface{})
```

Comment:
let's check for the second variable here and log a warning if !ok, wdyt?

Reply:
I added a log.